src/Parallel/PhaseControl/PhaseSelection.hpp

```diff
@@ -47,13 +47,13 @@ CREATE_HAS_TYPE_ALIAS(main_combine_method)
  * alias `phase_change_tags_and_combines` from the individual `PhaseChange`
  * objects, and can be aggregated from a list of `PhaseChange`s via
  * `PhaseControl::get_phase_change_tags`.
- * - `template <typename ParallelComponent> registration_list`: A metafunction,
- * must contain type alias `type` that is the list of registration actions that
- * are used to de-register and re-register when an element of
- * `ParallelComponent` is migrated (note: each item in these lists must be
- * actions that also specify `perform_deregistration` and `perform_registration`
- * static member functions that perform the de-registration and re-registration
- * when an element is migrated. See
+ * - `template <typename ParallelComponent> registration_list`: A metafunction
+ * that has the type alias, must contain type alias `type` that is the list of
```

Comment:
sorry, I wasn't clear before. "A metafunction that has the type alias `type` that is the list..."